### ISSUE TITLE
Typography - Justify

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#9]
**Requirement**: blocks of text should be aligned to either the left or the right, not justified.

**Why is this required?** People with dyslexia or some visual impairment find it much easier to read text when the spacing between words is consistent. Text that is justified can lead to “rivers of white” in the text which they may find particularly problematic.
